introduction.md

```diff
@@ -0,0 +1 @@
+":) I am Anto from BiH hope I can find some nerd friends"
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.